lib/bigint-buffer.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
```

Comment:
Also, it would be nice to use `readBigInt64LE` & `writeBigInt64LE` for Node v12 or higher and use existing code as a polyfill for previous versions. This way it would be much easier to remove this when Node v10 reaches end-of-life.